### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by andyOlarte514**
## Summary
- Profile data

### Android
- Pending

### iOS
- 
image

## Trello Card
- https://trello.com/c/9rYEMRr5/6-profile

### CODE REVIEW

File: src/app/screens/Profile/i18n.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import i18next from 'i18next';
+
+i18next.addResources('es', 'PROFILE', {
+  LOGOUT: 'Cerrar sesión',
```
Comment (santiagorobra) [inline src/app/screens/Profile/i18n.ts:4]: remember everything in english

**Comment by santiagorobra** [comment]
the android screenshots were missing

File: src/app/screens/Auth/screens/Login/i18n.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import i18next from 'i18next';
 
-i18next.addResources('es', 'LOGIN', {
+i18next.addResources('en', 'LOGIN', {
   MAIL: 'Email',
   MAIL_PLACEHOLDER: 'Ej: email@dominio.com',
```
Comment (santiagorobra) [inline src/app/screens/Auth/screens/Login/i18n.ts:5]: Ej?

File: src/app/screens/Profile/i18n.ts (hunk 1 above)
Comment (santiagorobra) [reply src/app/screens/Profile/i18n.ts:4]: missing

**Review by santiagorobra — APPROVED** [review]

File: src/app/components/WithHeader/index.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        )}
+      </View>
+      <View style={styles.contentContainer}>
+        <View style={styles.borderContainer}>
+          <View style={[styles.border]} />
+        </View>
+        <View style={[styles.childrenContainer, withAvatar && styles.longChildrenContainer]}>{children}</View>
+        {withAvatar && (
+          <View style={styles.avatarContainer}>
+            <Image source={userIcon} resizeMode="contain" style={styles.avatar} />
```
Comment (john07289) [inline src/app/components/WithHeader/index.tsx:39]: I think that this source should be dynamic

File: src/app/screens/Profile/components/ProfileItem/index.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { View, TouchableOpacity, ImageSourcePropType, Image } from 'react-native';
+import CustomText from '@components/CustomText';
+import iconArrow from '@assets/Profile/ic_general_row-arrow.png';
```
Comment (john07289) [inline src/app/screens/Profile/components/ProfileItem/index.tsx:4]: change this to icGeneralRowArrow use camelcase

File: src/app/screens/Profile/index.tsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import i18next from 'i18next';
+import { Keyboard, ScrollView, TouchableOpacity, View } from 'react-native';
```
Comment (john07289) [inline src/app/screens/Profile/index.tsx:3]: move this to second line

**Pull request merged**